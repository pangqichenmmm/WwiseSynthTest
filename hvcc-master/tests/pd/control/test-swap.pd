#N canvas 633 240 244 334 10;
#X msg 41 130 bang;
#X obj 41 52 loadbang;
#X obj 41 82 t b b b;
#X msg 80 130 3;
#X msg 86 185 4;
#X obj 41 275 print A;
#X obj 86 245 print B;
#X obj 41 215 swap 6.5;
#X obj 129 213 swap;
#X connect 0 0 7 0;
#X connect 1 0 2 0;
#X connect 2 0 0 0;
#X connect 2 1 4 0;
#X connect 2 2 3 0;
#X connect 3 0 7 0;
#X connect 4 0 7 1;
#X connect 7 0 5 0;
#X connect 7 1 6 0;
