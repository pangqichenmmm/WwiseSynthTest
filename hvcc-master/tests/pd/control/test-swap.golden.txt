[@ 0.000] B: 3
[@ 0.000] A: 6.5
[@ 0.000] B: 3
[@ 0.000] A: 4
